### PR TITLE
Update to Tunnel FAQ

### PR DESCRIPTION
**Pull request by abelinkinbio**

### CODE REVIEW

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ Yes. With [Named Tunnels](https://blog.cloudflare.com/argo-tunnels-that-live-for
 
 Yes. Cloudflare Tunnel has full support for Websockets.
 
+## ​Does Cloudflare Tunnel support gRPC?
+
+No. Cloudflare Tunnel does not support gRPC through public hostname deployments.
```
Comment (nmldiegues) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]: Let's say it works through private networking.
Add a link to https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/private-net/ to clarify what it is

**Review by nmldiegues — APPROVED** [review]

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md (hunk 1 above)
Comment (abelinkinbio) [reply content/cloudflare-one/faq/cloudflare-tunnels-faq.md]: Good call. Will do 👍

Comment (ranbel) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
Yes. Cloudflare Tunnel supports gRPC for services within a [private network](/cloudflare-one/connections/connect-apps/private-net/). Public hostname deployments are not supported at this time.
```

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 2]
```diff
@@ -55,20 +59,24 @@ No. When using Cloudflare Tunnel, all requests to the origin are made internally
 
 To log external visitor IPs, you will need to [configure an alternative method](https://support.cloudflare.com/hc/en-us/articles/200170786-Restoring-original-visitor-IPs-Logging-visitor-IP-addresses-with-mod-cloudflare-).
 
-## Why does the name "warp" appear in some legacy materials?
+## Why does the name "warp" and "argo" appear in some legacy materials?
 
-Cloudflare Tunnel was previously named Warp during the beta phase. As Warp was added to the Argo product family, we changed the name to match.
+Cloudflare Tunnel was previously named Warp during the beta phase. As Warp was added to the Argo product family, we changed the name to Argo Tunnel to match. Once we no longer required users to purchase Argo to create Tunnels, we changed the name to Cloudflare Tunnel to match. This one is here to stay.
```
Comment (ranbel) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
Cloudflare Tunnel was previously named Warp during the beta phase. As Warp was added to the Argo product family, we changed the name to Argo Tunnel to match. Once we no longer required users to purchase Argo to create Tunnels, we renamed Argo Tunnel to Cloudflare Tunnel.
```

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 3]
```diff
@@ -55,20 +59,24 @@ No. When using Cloudflare Tunnel, all requests to the origin are made internally
...
 
 ## How can I troubleshoot a Tunnel that was configured from the Zero Trust dashboard?
 
-### Ensure that only one Tunnel is installed
+### Ensure that only one instance of cloudflared is installed as a service
```
Comment (ranbel) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
### Ensure that only one instance of `cloudflared` is installed as a service
```

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 4]
```diff
@@ -55,20 +59,24 @@ No. When using Cloudflare Tunnel, all requests to the origin are made internally
...
 
-If you are unable to create a Tunnel using the installation script ("cloudflared service is already installed"), ensure that no other Tunnels are running as a service on this machine. Only a single Tunnel may run as a service on any given machine. Instead, we recommend adding additional routes to your existing Tunnel.  Alternatively, you can run `sudo cloudflared service uninstall` to uninstall the Tunnel.
+If you are unable to create a Tunnel using the installation script ("cloudflared service is already installed"), ensure that no other cloudflared instances are running as a service on this machine. Only a single instance of cloudflared may run as a service on any given machine. Instead, we recommend adding additional routes to your existing Tunnel.  Alternatively, you can run `sudo cloudflared service uninstall` to uninstall cloudflared.
```
Comment (ranbel) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
If you are unable to create a Tunnel using the installation script ("cloudflared service is already installed"), ensure that no other `cloudflared` instances are running as a service on this machine. Only a single instance of `cloudflared` may run as a service on any given machine. Instead, we recommend adding additional routes to your existing Tunnel.  Alternatively, you can run `sudo cloudflared service uninstall` to uninstall `cloudflared`.
```

File: content/cloudflare-one/faq/cloudflare-tunnels-faq.md [hunk 5]
```diff
@@ -55,20 +59,24 @@ No. When using Cloudflare Tunnel, all requests to the origin are made internally
...
 
 ### Check your DNS records
 
 If you are unable to save your Tunnel's public hostname ("An A, AAAA, or CNAME record with that host already exists"), choose a different hostname or delete the existing DNS record. [Check the DNS records](/dns/manage-dns-records/how-to/create-dns-records/) for your domain from the [Cloudflare dashboard](https://dash.cloudflare.com).
 
+### Run Tunnel with debug logging
+
+If you created a Cloudflare Tunnel from the dashboard, the tunnel runs as a service on your OS. You can modify the Cloudflare Tunnel service with one or more configuration options. Learn more in our the [Remote Management]([url](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/configuration/remote-management/)) section.
```
Comment (ranbel) [inline content/cloudflare-one/faq/cloudflare-tunnels-faq.md]:
```suggestion
If you created a Cloudflare Tunnel from the Zero Trust dashboard, the tunnel runs as a service on your OS. You can modify the Cloudflare Tunnel service with one or more configuration options. To learn more, refer to our [remote management](/cloudflare-one/connections/connect-apps/configuration/remote-management/) documentation. 
```

**Pull request merged**